<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memWindow</class>
 <widget class="QWidget" name="memWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>CPU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
        <widget class="QProgressBar" name="progressBar_CPU">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>431</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CPU：</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>内存和交换分区</string>
        </property>
        <widget class="QProgressBar" name="progressBar_RAM">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>431</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>内存：</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar_SWAP">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>140</y>
           <width>431</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>交换：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>80</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Used：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>80</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Left：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>80</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Total：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>170</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Total：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>170</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Left：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>170</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Used：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_RAM_USED">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>80</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_RAM_LEFT">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>80</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_RAM_TOTAL">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>80</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_SWAP_USED">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>170</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_SWAP_LEFT">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>170</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_SWAP_TOTAL">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>170</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
